ui/src/tasks/reducers/tasks.test.ts

```diff
@@ -0,0 +1,50 @@
+import tasksReducer, {
+  defaultState,
+  defaultTaskOptions,
+} from 'src/tasks/reducers'
+import {setTaskOption} from 'src/tasks/actions'
+import {TaskSchedule} from 'src/utils/taskOptionsToFluxScript'
+
+describe('tasksReducer', () => {
+  it('clears the cron property from the task options when interval is selected', () => {
```

Comment:
these tests could have a another describe for the action these reducers are responding to.  i.e."

```
describe('tasksReducer', () => {
  describe('setTaskOption', () => {
    ... setTaskOption tests here
   })
})
```